app/domain/errors.rb

```diff
@@ -338,6 +343,26 @@ module AuthnAzure
         code: "CONJ00054E"
       )
     end
+
+    module AuthnGce
+
+      InvalidAudience = ::Util::TrackableErrorClass.new(
+        msg:  "'audience' token claim {0} is invalid. The format should be " \
+              "'conjur/<account_name>/<host_id>'",
+        code: "CONJ00067E"
+      )
+
+      TokenClaimNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
+        msg:  "Claim '{0-claim-name}' not found or empty in token. " \
+              "Verify you request the token using 'format=full'",
```

Comment:
Make sure that you requested the token using ...

or 
Make sure to request the token using...

depending on if it's already happened, or something that you are "instructing them to do next"